src/client/Microsoft.Identity.Client/Platforms/net45/Http/DnsSensitiveClientHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Http;
+
+namespace Microsoft.Identity.Client.Platforms.net45.Http
+{
+    internal class DnsSensitiveClientHandler : DelegatingHandler
```

Comment:
>DnsSensitiveClientHandler [](start = 19, length = 25)

isn't this also needed for netcore?

Reply:
ah, you are setting it just not using this helper class...

---
In reply to: [446375812](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1906#discussion_r446375812) [](ancestors = 446375812)

Reply:
On .net core they've left out `ServicePointManager.FindServicePoint`. Luckily they introduced a mechanism in .net core 2.1 and MSAL targets exactly .net core 2.1 ... That's why the .net core implementation is much simpler.